bower.json

```diff
@@ -62,5 +62,8 @@
         "build/jsfeat.js"
       ]
     }
+  },
+  "resolutions": {
```

Comment:
Yea, let's do that - I'll try to repoduce. But `<dcjs>` should be moved to `<dependencies>` if it doesn't break anything.